packages/components/spec/components/date-time/DateTime.spec.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import { DateTime } from '../../../src/components';
+import { DateTimeFormat } from '../../../src/components/date-time/interfaces';
+
+describe('DateTime', () => {
+  const testDate = new Date(2024, 11, 30, 17, 30, 15);
+  const testISOString = testDate.toISOString();
```

Comment:
Nitpicking. Feel free to edit or not, no big deal.

But this should be declared in the test that uses it. No point in having it declared for all test files.